l10n_th_withholding_tax_cert/models/withholding_tax_cert.py

```diff
@@ -29,11 +29,10 @@
 
 class WithholdingTaxCert(models.Model):
     _name = "withholding.tax.cert"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
     _description = "Withholding Tax Certificate"
 
-    name = fields.Char(
-        string="Number", readonly=True, related="payment_id.name", store=True
-    )
+    name = fields.Char(string="Number")
```

Comment:
Should this be a computed field, with readonly=False? (so it behave like onchange)